contracts/node.sh

```diff
@@ -51,7 +51,7 @@ main()
 
         FORK=true npm run copy-interface-artifacts
         FORK=true npx hardhat fund --amount 100000 --network localhost --accountsfromenv true &
-        
+
```

Comment:
Did the linter do that? if not please leave as is so that this file does not show in the diff

Reply:
fixed